app/js/openmrs.js

```diff
@@ -0,0 +1,119 @@
+angular.module('openmrs.resources', [ 'ngResource' ])
```

Comment:
Any reason why you named the module openmrs.resources? I would rather we name it just openmrs. We can divide it in more modules later on, but I'd like other apps to just be able to require one openmrs module and have all utils available.